packages/toast/src/ToastDefault.tsx

```diff
@@ -1,7 +1,10 @@
-import { toast, ToastOptions } from 'react-toastify'
 import { ReactText } from 'react'
+import { toast, ToastOptions, ToastContent } from 'react-toastify'
 import { TOASTS_DEFAULT_OPTIONS } from './toastsDefaultOptions'
 
-export function ToastDefault(text: string, options: ToastOptions): ReactText {
-  return toast(text, { ...TOASTS_DEFAULT_OPTIONS, ...options })
+export function ToastDefault(
+  content: ToastContent,
+  options?: ToastOptions
+): ReactText {
+  return toast(content, { ...TOASTS_DEFAULT_OPTIONS, ...options })
```

Comment:
If options object is empty, what would be destructured here?

Reply:
options can be undefined. so will get only TOASTS_DEFAULT_OPTIONS